classes/journal/JournalDAO.inc.php

```diff
@@ -204,7 +204,7 @@ function deleteJournalById($journalId) {
 	 * @param $enabledOnly boolean True iff only enabled jourals wanted
 	 * @param $rangeInfo object optional
 	 * @param $sortBy JOURNAL_FIELD_... optional sorting parameter
-	 * @param $searchField JOURNAL_FIELD_... optional filter parameter
+	 * @param $searchField JOURNAL_FIELD_... optional filter parameter; DEPRECATED - use getBySetting() instead
```

Comment:
Since the core code doesn't reference this parameter, I wanted to dissuade other plugins in development from choosing this function instead of getBySetting().  How about I remove the trigger_error() but keep the `use getBySetting() instead` in the documention?